### PR TITLE
Completed css flexbox exercise.

### PR DESCRIPTION
**Pull request by randygerada**
Screen Shot 2022-10-13 at 9 36 09 AM
Screen Shot 2022-10-13 at 9 36 17 AM
Screen Shot 2022-10-13 at 9 36 24 AM
Screen Shot 2022-10-13 at 9 36 37 AM

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      min-height: 100vh;
+      align-items: center;
+      justify-content: center;
+    }
+
```
Comment (BrettAlbright) [inline css-flexbox/align-items.html:18]: you are missing the CSS for the card wrapper here.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one update needed.

**Comment by randygerada** [comment]
Screen Shot 2022-10-13 at 11 32 59 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**